zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/TableColumns.kt

```diff
@@ -0,0 +1,60 @@
+package org.hexworks.zircon.api
+
+import org.hexworks.cobalt.databinding.api.binding.bindTransform
+import org.hexworks.cobalt.databinding.api.property.Property
+import org.hexworks.cobalt.databinding.api.value.ObservableValue
+import org.hexworks.zircon.api.component.Icon
+import org.hexworks.zircon.api.component.Label
+import org.hexworks.zircon.internal.fragment.impl.table.TableColumn
+
+/**
+ * This object provides different presets for [TableColumn]s.
+ */
+object TableColumns {
+
+    /**
+     * Creates a [TableColumn] that represents it's cell values with a [Label]. The text of the label
+     * is the toString() of the value provided by [valueAccessor].
+     *
+     * @param name the name of the column to be used as header
+     * @param width the width of the column
+     * @param valueAccessor returns the value to be used in each cell of the column. It should preferable be a
+     *  [String] or similar primitive type so that calling toString on it does not have strange side effects.
+     *  When the value returned is an [ObservableValue] or a [Property] the textProperty of the label
+     *  will update accordingly.
+     */
+    fun <M : Any, V : Any> textColumn(name: String, width: Int, valueAccessor: (M) -> V): TableColumn<M, V, Label> =
```

Comment:
Would it make sense to have a version of `textColumn` that explicitly has an `ObservableValue` as its `valueAccessor`? Just for good DX.

Reply:
I am not fully satisfied with the name, but added `TableColumns.textColumnObservable()` with 79379695